modules/core/src/lib/layers/editable-layer.js

```diff
@@ -202,7 +202,8 @@ export default class EditableLayer extends CompositeLayer {
       mode: 'query',
       layers: [this.props.id],
       radius: 10,
-      viewports: [this.context.viewport]
+      viewports: [this.context.viewport],
+      depth: 2
```

Comment:
@supersonicclay I noticed the same behavior as before, where the picked features alternated between including the intermediate edit handle, and thus produced a pulsating effect. Needed to still add the depth (and upgrade deck.gl) to resolve.

Reply:
Gotcha, we'll hold off until we can upgrade deck.gl to include the fix:
https://github.com/uber/deck.gl/pull/2271